GitUI/Properties/Resources.Designer.cs

```diff
@@ -254,6 +254,16 @@ internal class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized resource of type System.Drawing.Bitmap.
+        /// </summary>
+        internal static System.Drawing.Bitmap Settings {
```

Comment:
We already have this icon, though it is probably under a different name:
![image](https://user-images.githubusercontent.com/4403806/63489655-5fff3500-c4bb-11e9-8a1a-daa8c7fce4df.png)

Please re-use it instead of adding a new resource

Reply:
Thanks. I didn't know of the `Images` resources. I was miss leaded by the fact that there are images in the `Resources` resources.